src/Modules/OrchardCore.Commerce/Startup.cs

```diff
@@ -170,6 +170,9 @@ public override void ConfigureServices(IServiceCollection services)
         // Page
         services.AddScoped<IDataMigration, PageMigrations>();
 
+        // Checkout
+        services.AddScoped<ICheckoutEvents, InventoryCheckoutEvents>();
```

Comment:
We only need inventory events if the `OrchardCore.Commerce.Inventory` feature is enabled. So please move this into `InventoryStartup` class below.